tooling-tests/src/test/resources/test-src/output/tool_test_generate_13/target/persist_db_scripts.sql

```diff
@@ -1,27 +1,28 @@
 DROP TABLE IF EXISTS MultipleAssociations;
-CREATE TABLE MultipleAssociations (
+
+DROP TABLE IF EXISTS Profiles;
```

Comment:
Is this needed for tests? Else lets update gitignore file to ignore files under target directory

Reply:
Since we are compiling the project to detect syntax and semantic errors, sql file is automatically generated. This is required to assert the equality of the actual and expected content.